src/base/noscript.scss

```diff
@@ -8,6 +8,12 @@
 	display: block !important;
 }
 
+@media screen {
+	%noscript-screen-display-block-important {
```

Comment:
@LaurentGoderre Here's a few Codepen examples. Both generate the same CSS output.
* [Placeholder selector inside a media query (i.e. what this PR is currently doing)](https://codepen.io/anon/pen/gggmJa)
* [Placeholder selector containing a media query](https://codepen.io/anon/pen/GrrmJo)

I wouldn't mind moving the media query inside ``%noscript-display-block-important`` if you'd prefer.